### PR TITLE
[zh-cn]: create the translation of `browserAction.getBadgeTextColor()` method

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+slug: Mozilla/Add-ons/WebExtensions/API/browserAction/getBadgeTextColor
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+获取浏览器操作徽章的文本颜色。
+
+从 Firefox 63 开始，除非使用 {{WebExtAPIRef("browserAction.setBadgeTextColor()")}} 显式设置了徽章文本颜色，否则徽章文本颜色将自动设置为黑色或白色，以使其与指定的徽章背景颜色最大对比度。例如，如果你将徽章背景颜色设置为白色，则默认的徽章文本颜色将设置为黑色，反之亦然。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md:12]:
```suggestion
从 Firefox 63 版本开始，除非使用 {{WebExtAPIRef("browserAction.setBadgeTextColor()")}} 显式设置了徽章文本颜色，否则徽章文本颜色将自动设置为黑色或白色，以使其与指定的徽章背景颜色形成最大对比度。例如，如果你将徽章背景颜色设置为白色，则默认的徽章文本颜色将设置为黑色，反之亦然。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      - : `integer`。指定从中获取徽章文本颜色的窗口。
+
+    <!---->
+
+- 如果同时提供了 `windowId` 和 `tabId`，则函数失败。
+- 如果 `windowId` 和 `tabId` 都省略，则返回全局徽章文本颜色。
+
+### 返回值
+
+一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现包含检索到的颜色的{{WebExtAPIRef('browserAction.ColorArray')}}。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md:44]:
```suggestion
一个 [`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，其会兑现包含检索到的颜色的 {{WebExtAPIRef('browserAction.ColorArray')}}。
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]

File: files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+- details
+
+  - : `object`。
+
+    - `tabId` {{optional_inline}}
+      - : `integer`。指定从中获取徽章文本颜色的标签页。
+    - `windowId` {{optional_inline}}
+      - : `integer`。指定从中获取徽章文本颜色的窗口。
+
+    <!---->
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/browseraction/getbadgetextcolor/index.md:37]: image

```suggestion
<!---->
```

**Comment by yin1999** [comment]
请更新一下内容

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**